aptos-move/framework/aptos-experimental/sources/trading/order_book/active_order_book.move

```diff
@@ -62,7 +62,7 @@ module aptos_experimental::active_order_book {
         // so that max_key never changes, and doesn't create conflict.
         ActiveOrderBook::V1 {
             buys: new_default_big_ordered_map(),
-            sells: new_default_big_ordered_map(),
```

Comment:
why removing this? much cleaner to have comma at the end of every line (I think that is a style guidance)

Reply:
The formatter actually removed this.